### PR TITLE
Added example topologies and updated docs wrto the same

### PR DESCRIPTION
**Pull request by samvarankashyap**
Updated docs 
Added source of example topologies

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,5 +10,6 @@ boto==2.40.0
 apache-libcloud==0.20.1
 sphinx==1.4.4
 jsonschema==2.5.1
-libvirt-python==2.1.0
```
Comment (herlo) [inline requirements.txt]: How is this going to affect a libvirt-xml strategy?

File: docs/source/gcloud_topo.rst [hunk 2]
```diff
@@ -8,14 +8,51 @@ Gcloud Topologies
 Google Cloud Topologies
 ```````````````````````
 
-Google cloud topologies . 
+.. code-block:: bash
```
Comment (herlo) [inline docs/source/gcloud_topo.rst]: Would this be better with code-block:: yaml?

http://symfony.com/doc/current/contributing/documentation/format.html#syntax-highlighting

File: docs/source/gcloud_topo.rst [hunk 3]
```diff
@@ -54,5 +54,6 @@ Google Cloud Topologies
 
 .. note::
 
-   
+  Source of the above mentioned examples can be fount at following link:
```
Comment (herlo) [inline docs/source/gcloud_topo.rst]: Found is misspelled here.

File: docs/source/gcloud_topo.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+==================
+
+.. contents:: Topics
+
+.. _gcloud_topologies:
+
+Google Cloud Topologies
+```````````````````````
+
+.. code-block:: bash
```
Comment (herlo) [inline docs/source/gcloud_topo.rst]: code-block:: yaml

File: requirements.txt (hunk 1 above)
Comment (samvarankashyap) [inline requirements.txt]: @herlo : Readthedocs is fail to build linchpin docs when libvirt-python is mentioned in requirement.txt , I removed the package from requirement.txt in oder to make successful build in read the docs .

File: docs/source/gcloud_topo.rst (hunk 2 above)
Comment (samvarankashyap) [inline docs/source/gcloud_topo.rst]: @herlo : my bad , I maintained code block yaml through out . Just missed out in gcloud topology.

File: docs/source/gcloud_topo.rst (hunk 3 above)
Comment (samvarankashyap) [inline docs/source/gcloud_topo.rst]: The repercussions of using a vi editor without a spell check.
fixing it with followup commit .

**Pull request merged**